tests/python/frontend/onnx/test_forward.py

```diff
@@ -4640,6 +4640,48 @@ def repeat(N, D):
     )
 
 
+def test_LinearRegressor():
+    def verify_LinearRegressor(a_shape, c_shape, i_shape, targets=1, batch=1):
+        a_array = np.random.uniform(size=a_shape).astype("float32")
+        out_shape = (batch, targets)
+
+        coefficients = np.random.uniform(size=c_shape).astype("float32")
+        intercepts = np.random.uniform(size=i_shape).astype("float32")
+
+        mul_node = helper.make_node(
+            "LinearRegressor",
+            ["a"],
+            ["out"],
+            coefficients=coefficients,
+            intercepts=intercepts,
+            targets=targets,
+            domain="ai.onnx.ml",
+        )
```

Comment:
I was able to reproduce the error using onnx-1.6.0 (which is used in test image tlcpack/ci-gpu:v0.75)
Error: `ValueError: You passed in an iterable attribute but I cannot figure out its applicable type.`
Code to reproduce the error:
```
# python3
from onnx import helper
import numpy as np

a_shape = (2, 10)
c_shape = (10)
i_shape = (1)
targets=1
batch=2
a_array = np.random.uniform(size=a_shape).astype("float32")
out_shape = (batch, targets)
coefficients = np.random.uniform(size=c_shape).astype("float32")
intercepts = np.random.uniform(size=i_shape).astype("float32")

mul_node = helper.make_node(
    "LinearRegressor",
    ["a"],
    ["out"],
    coefficients=coefficients,
    intercepts=intercepts,
    targets=targets,
    domain="ai.onnx.ml",
)
```

Reply:
to determine coefficients values types onnx
1.6.0 uses `isinstance(v, float)`
1.8.1 uses `isinstance(v, numbers.Real)`
So, you'd better to use plain Python arraya instead of Numpy arrays when calling `onnx.helper.make_node`

Possible fix:
```
>>> mul_node = helper.make_node(
...     "LinearRegressor",
...     ["a"],
...     ["out"],
...     coefficients=coefficients.tolist(),
...     intercepts=intercepts.tolist(),
...     targets=targets,
...     domain="ai.onnx.ml",
... )
```